misk-action-scopes/src/main/kotlin/misk/scope/RealActionScoped.kt

```diff
@@ -8,4 +8,6 @@ internal class RealActionScoped<T> @Inject internal constructor(
   val scope: ActionScope
 ) : ActionScoped<T> {
   override fun get(): T = scope.get(key)
+
+  override fun getIfInScope(): T? = if (scope.inScope()) get() else null
```

Comment:
I wonder whether getOrNull would be more idiomatic with Kotlin std lib naming of methods given the get in this case throws if not within a scope I think?

Reply:
`getOrNull` is the name I initially also thought of. But this method only returns null if "not in scope", but throws an exception when it delegates to `get` if the key is not found. That'd the existing behaviour we need, and `getOrNull` didn't seemed like a misleading name for that.